### ISSUE TITLE
Incorrect aria-label is defined for the 'Collapsed (+), Expand(-) icons button' present under 'Navigation' popup.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#102]
User Experience:

User depending on screen reader might get affected as aria label for the control is incorrectly defined and screen reader will never be able to pass the information correctly due to this user will not be able to understand the purpose of the control.

Note: User credentials should NOT be included in the bug.

 

Repro steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode.

4. Navigate to the Left nav and select Navigation button.

5. Navigate to the various controls of the "Navigation" window like main menu, home, API details, products etc.

5. Open developer tool using F12 key and navigate to collapsed (+), expand(-) icons buttons.

6. Verify the issue.

 

Actual Result:

Incorrect aria-label is defined for the collapsed (+), expand(-) icons buttons, screen reader passes irrelevant information as “Toggle nested items” button for every button of this type in same popup, also its not descriptive.
 

Expected Result:

Aria- label should be defined for all ‘collapsed (+), expand(-) icons’ buttons uniquely as ‘Main menu’ expand collapsed button, ‘Authenticated user menu’ expand collapsed button etc.'. 

Note:

1. Similar issue exist with Chrome.
 

MAS Reference:

MAS 4.1.2 - Name, Role, Value (40B)

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.
